docs/framework/contributing/code-style.md

```diff
@@ -1062,3 +1063,75 @@ import ClassicEditor from '@ckeditor/ckeditor5-build-classic/src/ckeditor';
 ```
 
 [History of the change.](https://github.com/ckeditor/ckeditor5/issues/13689)
+
+### Declaring module augmentation for the core package: `allow-declare-module-only-in-augmentation-file`
```

Comment:
All rule names in this guide are prefixed with `ckeditor5-rules/`:

```suggestion
### Declaring module augmentation for the core package: `ckeditor5-rules/allow-declare-module-only-in-augmentation-file`
```

If you will align your changes here to the existing format, please remember also to adjust the `meta.docs.url` property for the changed rules in `eslint-plugin-ckeditor5-rules` package.